sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/ContactService.java

```diff
@@ -68,8 +68,8 @@ public Contact buildGeneratedContact() {
         .reportDate(LocalDate.now().minusDays(random.nextInt(10)))
         .diseaseOfSourceCase("COVID-19")
         .caseIdInExternalSystem(UUID.randomUUID().toString())
-        .dateOfFirstContact(LocalDate.now().minusDays(5))
-        .dateOfLastContact(LocalDate.now().minusDays(3))
+        .dateOfFirstContact(LocalDate.now().minusDays(15))
+        .dateOfLastContact(LocalDate.now().minusDays(13))
```

Comment:
why we change this? this method is used among the framework, and I need to understand how safe is this change

Reply:
Hi, as you can see in line 68 report date is a random from range 0-9 and sometimes the test was failing because date of first contact was after the date of report which is a logical error for SORMAS.